### DISCUSSION TITLE
save node config?

### DISCUSSION THREAD

**Discussion by illogique** [alicevision/Meshroom#1715] [Q&A]
hello i follow the guide to use non-nvidia card here https://github.com/alicevision/meshroom/wiki/Draft-Meshing
but i have to do it every time i launch meshroom.
how can i save the config?

**Comment by adam-bielanski** [comment]
You can try to run it on Google Colab cloud too. The disadvantage is the 12 h limit that limits that tool for any more complex sets of pictures.

⁣Pobierz aplikację TypeApp dla systemu Android ​

W dniu 20 cze 2022, 17:38, o 17:38, użytkownik illogique ***@***.***> napisał:
